tasks/zip.js

```diff
@@ -31,8 +34,31 @@ module.exports = function( gulp ) {
 			commonZipInclude = commonZipInclude.map( fileName => 'common/' + fileName );
 		}
 
+		// Only for .org glotpress we do this check.
+		if ( 'https://translate.wordpress.org' === json._glotPressUrl ) {
```

Comment:
Is this not a bit weak? Would a regular expression test here provide more robustness?